### PR TITLE
Fm/features/finish pagination

### PR DESCRIPTION
**Pull request by frankmangone**

### CODE REVIEW

File: src/entities/calendar/calendar.model.ts [hunk 1]
```diff
@@ -35,14 +42,32 @@ export class Calendar extends Utils.BaseModel {
...
+  // *
+  // Relations
+  // *
 
   @GQL.Field(() => Number)
   @DB.Prop({ type: Number, default: 0 })
   followerCounter: number
+
+  @GQL.Field(() => [CalendarEvent])
+  @Utils.OneToMany()
```
Comment (gastonmorixe) [inline src/entities/calendar/calendar.model.ts]: 🚨

File: src/entities/calendar/calendar.resolver.ts [hunk 2]
```diff
@@ -53,4 +60,28 @@ export class CalendarResolver {
...
+    const filters = { calendar: calendar._id }
+    return this.calendarEventService.list(filters, paginationArgs)
+  }
+
+  @ResolveField()
+  async usersFollowerConnection(
+    @Parent() calendar: CalendarDocument,
+    @Args() paginationArgs: PaginationArgs,
+  ) {
+    const filters = { resourceId: calendar._id.toString() }
```
Comment (gastonmorixe) [inline src/entities/calendar/calendar.resolver.ts:82]: 👮 add resource name

```suggestion
    const filters = { resourceId: calendar._id.toString(), resourceName: Calendar.name }
```

File: src/entities/user/user.model.ts [hunk 3]
```diff
@@ -2,11 +2,19 @@ import * as DB from '@nestjs/mongoose' // { Prop, Schema, SchemaFactory }
 import * as GQL from '@nestjs/graphql' // { Field, ObjectType, ID }
 import * as V from 'class-validator' // { Prop, Schema, SchemaFactory }
 import mongoose from 'mongoose'
-
+import autopopulate from 'mongoose-autopopulate'
+import { Paginated, PaginatedType, withCursor } from '../../utils/Pagination'
```
Comment (gastonmorixe) [inline src/entities/user/user.model.ts:6]: ☢️

**Review by gastonmorixe — CHANGES_REQUESTED** [review]
11

File: src/entities/calendar/calendar.resolver.ts (hunk 2 above)
Comment (frankmangone) [reply src/entities/calendar/calendar.resolver.ts:82]: No, it's not necessary in this case. The only collection that needs that extra filter is Follower

File: src/entities/event/event.model.ts [hunk 4]
```diff
@@ -1,11 +1,16 @@
 import * as DB from '@nestjs/mongoose' // { Prop, Schema, SchemaFactory }
 import * as GQL from '@nestjs/graphql' // { Field, ObjectType, ID }
 import mongoose from 'mongoose'
+import autopopulate from 'mongoose-autopopulate'
+import * as Utils from '../../utils/Model'
```
Comment (gastonmorixe) [inline src/entities/event/event.model.ts:5]: ⚠️ 🧐

**Review by gastonmorixe — APPROVED** [review]

**Pull request merged**